commands/health.nicole

```diff
@@ -59,7 +59,7 @@ function cmd_logs_show {
     if [[ -f "${file}" ]] && [[ "$(realpath "${link}")" = "${file}" ]]; then
       # Should exec fail, the script will abort due to `set -e` in effect
       DREPORT_INCLUDE="/usr/share/cli/.include" \
-      TIME_STAMP=$(date -u) exec "${file}"
+      TIME_STAMP="date -u" exec "${file}"
```

Comment:
The question is: do we actually need any time stamp at all when printing error messages to the user?

Reply:
This line restores the original `dreport`'s behavior.
If you think that we should change it, we can discuss your proposal and make it as a standalone change.